src/main/java/com/google/cloud/tools/appengine/AppEngineDescriptor.java

```diff
@@ -65,31 +60,32 @@ public static AppEngineDescriptor parse(InputStream in) throws IOException, SAXE
   }
 
   /**
-   * @return project ID from the &lt;application&gt; element of the appengine-web.xml or null
-   *         if it is missing
+   * Returns project ID from the &lt;application&gt; element of the appengine-web.xml or null if it
```

Comment:
Is this an autofix? Seems strange.

Reply:
No, it's a manual fix to comply with javadoc style : see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment